tests/async/tasync_forward.nim

```diff
@@ -7,3 +7,12 @@ proc foo {.async.}
 
 proc foo {.async.} =
   discard
+
+# With additional pragmas:
+proc bar {.async, cdecl.}
```

Comment:
It's not the default. The sample will not compile if you remove cdecl from bar declaration.